### PR TITLE
EL-3289 - Popover Improvements

### PR DESCRIPTION
**Pull request by ashley-hunter**
Related: https://github.houston.softwaregrp.net/caf/quantum/pull/586 & https://github.houston.softwaregrp.net/caf/ux-aspects-micro-focus/pull/302

Added support for `top-left`, `top-right`, `bottom-left` and `bottom-right` popover placements

### CODE REVIEW

**Comment by ashley-hunter** [comment]
Made the suggested changes, thanks

**Comment by ashley-hunter** [comment]
Thanks, Alignment now only uses `start`, `center` and `end`.

File: docs/app/pages/components/components-sections/popover/popover/popover.component.html [hunk 1]
```diff
@@ -45,16 +45,20 @@
         If set to <code>true</code> this will programmatically show the popover.
     </tr>
     <tr uxd-api-property name="placement" type="string" defaultValue="top">
-        Defines which direction the popover should appear. The possible values are 
+        Defines which direction the popover should appear relative to the trigger element. The possible values are
         <code>top</code>, <code>right</code>, <code>bottom</code> and <code>left</code>.
     </tr>
+    <tr uxd-api-property name="alignment" type="string" defaultValue="center">
+        Defines which the position of the arrow relative to the popover. The possible values are
```
Comment (donna-marijne) [inline docs/app/pages/components/components-sections/popover/popover/popover.component.html]: Typo "which the..."

**Comment by ashley-hunter** [comment]
Thanks, typo fixed

**Pull request merged**